### DISCUSSION TITLE
No Email Notifications

### DISCUSSION THREAD

**Discussion by feedjosiah** [semaphoreui/semaphore#1093] [Q&A]
I am having trouble getting email notifications. I have configured a Semaphore container to email using the following config:
```
{
  ...
  "email_sender": "semaphore@domain.com",
  "email_host": "172.17.0.1",
  "email_port": "25",
  ...
  "email_alert": true,
  ...
}
```
And Semaphore project has been set to allow alerts. Also, I have verified that emails can be sent from the container with `sendmail` command and using the same IP address shown in the config 172.17.0.1 (host IP on docker0 interface).

I've tailed the email log file (/etc/var/log/maillog) on the host and didn't see any activity from Semaphore but the `sendmail` test shows up. What am I missing?

**Comment by fiftin** [comment]
Looks like we have too many flags for alerts 😀 

You should check flag in project and user settings.

**Reply by feedjosiah** [reply]
I have those set already too.

Project
![image](https://user-images.githubusercontent.com/38047212/202912575-7dbe7f1e-8ca6-40c4-8393-b674b4606894.png)

User Settings
![image](https://user-images.githubusercontent.com/38047212/202912604-f570b79f-52ec-446d-855b-41d7d7431d4d.png)

Any other ideas? Thanks for helping me out with this.

**Comment by masterwishx** [comment]
i have also no alert in email

**Comment by masterwishx** [comment]
![image](https://github.com/user-attachments/assets/70ada9ae-3516-4ec9-bf1f-c7e78a71b93f)
****

**Reply by cdgredsis** [reply]
were you able to solve it?
